config/settings/prod.yml

```diff
@@ -0,0 +1,14 @@
+dfe_signin:
+  issuer: https://oidc.signin.education.gov.uk
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
+  profile: https://profile.signin.education.gov.uk
+  base_url: https://www2.publish-teacher-training-courses.service.gov.uk
+manage_backend:
+  base_url: https://api2.publish-teacher-training-courses.service.gov.uk
+manage_ui:
+  base_url: https://www.publish-teacher-training-courses.service.gov.uk
+search_ui:
+  base_url: https://www.find-postgraduate-teacher-training.service.gov.uk
+environment:
+  label: "Beta"
+  selector_name: "beta"
```

Comment:
https://github.com/DFE-Digital/manage-courses-frontend/blob/151d12ed7aa4918aef1c87e6ba6bfb2cab48d5d6/config/settings/new_prod.yml#L14

does casing matters?

Reply:
Don't know if case matters. But it should work as is because I copied it from production.yml file which is currently in use.